python/packages/microsoft_agents_m365copilot_beta/pyproject.toml

```diff
@@ -5,14 +5,14 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry]
 name = "microsoft-agents-m365copilot-beta"
 # x-release-please-start-version
-version = "1.0.0-preview.1"
+version = "1.0.0-preview.2"
```

Comment:
This should be managed via release-please

Reply:
Ack. Will apply the discussed changes in #84